webui/react/src/components/OverviewStats.module.scss

```diff
@@ -1,24 +1,7 @@
-.base {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
+.info {
+  font-size: 13px;
 
-  .title {
-    font-size: 12px;
-    font-weight: normal;
-  }
-  .info {
-    font-size: 13px;
-
-    & > :global(.ant-typography) {
-      margin: 0;
-
-      & small {
-        font-size: 12px;
-      }
-    }
-    &.clickable > :global(.ant-typography) {
-      color: var(--theme-ix-on-active);
-    }
+  &.clickable > :global(.ant-typography) {
+    color: var(--theme-ix-on-active);
   }
 }
```

Comment:
make sure this is necessary?

Reply:
the font-size rule was not being applied; removed it but kept this `.clickable` class for the time it's used for the best checkpoint